compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/MockDesugar.java

```diff
@@ -171,17 +176,48 @@ private BLangFunctionBody generateBody() {
     }
 
     private List<BLangStatement> generateStatements() {
-        List<BLangStatement> statements = new ArrayList<>();
-
+        // <MockFunctionObj>.functionToMock = (functionToMock);
+        BLangAssignment bLangAssignment =
+                ASTBuilderUtil.createAssignmentStmt(bLangPackage.pos, generateFieldBasedAccess(), generateRHSExpr());
         // BLangReturn Statement <retType> test:MockHandler(<MockFunctionObj>, [<args?>])
```

Comment:
Commented code?

Reply:
The commented part is a reference. It simply shows what code it is that we are trying to generate